### PR TITLE
chore: Nullable Profile

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
Allows returning null as profile. More info on why we are doing this change, at this PR:
https://github.com/vtex-sites/base.store/pull/418

### CODE REVIEW

File: packages/api/src/platforms/vtex/resolvers/query.ts [hunk 1]
```diff
@@ -158,14 +158,16 @@ export const Query = {
...
 
-    return {
-      id: profile?.id?.value ?? '',
-      email: profile?.email?.value ?? '',
-      givenName: profile?.firstName?.value ?? '',
-      familyName: profile?.lastName?.value ?? '',
-    }
+    return (
+      profile && {
+        id: profile.id,
```
Comment (lariciamota) [inline packages/api/src/platforms/vtex/resolvers/query.ts:166]: Why did you remove the `.value` from here?

Comment (tlgimenes) [reply packages/api/src/platforms/vtex/resolvers/query.ts:166]: nice catch

**Review by igorbrasileiro — APPROVED** [review]
